src/fibonacciBases.js

```diff
@@ -0,0 +1,47 @@
+function fibonacci(n) {
+  if (n === 0 || n === 1) {
+    return 1;
+  } else {
+    return fibonacci(n - 2) + fibonacci(n - 1);
+  }
+}
+
+function decToFibonnaci(number) {
+  let array = []
+  let fibonacciStack = []
+  let i = 1
+  let k = 0
+  let m = 0
+
+  while (number >= fibonacci(i)) {
+    i += 1
+    fibonacciStack.push(i - 1)
+    if (number - fibonacci(i - 1) !== 0) {
+      decToFibonnaci(number - fibonacci(i - 1))
+    } else {
+      while (k < fibonacciStack[0]) {
+        array.push(0)
+        k += 1
+      }
+      while (m < fibonacciStack.length) {
+        array[fibonacciStack[m] - 1] = 1
+        m += 1
+      }
+    }
+    return array.split("").reverse().join("")
```

Comment:
This might be your array problem. I'm sure because you dont have tests :P but… I don't think `split` is an array method. Its a string method.